core/rawdb/freezer.go

```diff
@@ -301,7 +301,32 @@ func (f *Freezer) TruncateHead(items uint64) (uint64, error) {
 	return oitems, nil
 }
 
-// TruncateTail discards any recent data below the provided threshold number.
+// TruncateTailBlocks discards all data below the provided threshold block number.
+// This is intended to be used with the chain freezer. Unlike TruncateTail, it
+// only truncates blocks and receipts, leaving the header table where it is.
+func (f *Freezer) TruncateTailBlocks(tailBlock uint64) (uint64, error) {
```

Comment:
Can we get rid of `TruncateTailBlocks` and integrate the `prunable` notion into `TruncateTail` instead?

Reply:
Ahh, that's what you mean there. I guess we could.